### PR TITLE
hetzner: add FIXME placeholders

### PR DESCRIPTION
**Pull request by asymmetric**
To make the necessary edits easier to find.

### CODE REVIEW

File: hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh [hunk 1]
```diff
@@ -160,8 +160,7 @@ set +u +x # sourcing this may refer to unset variables that we have no control o
 . $HOME/.nix-profile/etc/profile.d/nix.sh
 set -u -x
 
-# Keep in sync with `system.stateVersion` set below!
-# nix-channel --add https://nixos.org/channels/nixos-20.03 nixpkgs
```
Comment (asymmetric) [inline hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh:164]: I also removed this duplicate line,

**Review by happysalada — APPROVED** [review]
It looks good to me.
I don't have a very strong opinion on this either way.

**Comment by asymmetric** [comment]
I actually don't understand if the mdadm `homehost` should also be kept in sync with the `hostname` - if so, then we should add `FIXME`s there too.

**Comment by happysalada** [comment]
I think so, but I have never gone fully through that tutorial.
@nh2 should know, when he has a moment to review.

**Comment by asymmetric** [comment]
I think I understand the comments better now, `homehost` is supposed to stay unchanged, since it was first set by Hetzner to `hetzner`.

@nh2 true story?

**Comment by happysalada** [comment]
After checking, the default hostname on a hetzner machine is not hetzner
you can check this by doing `hostname`
I think this value should be replaced with your default hostname, for example mine is `Ubuntu-2010-groovy-64-minimal`
Note that it should not be the hostname you will set in the future, but the current hostname. (that is my understanding)

**Comment by happysalada** [comment]
Also just noticed, but we can probably update the stable to 20.09 instead of 03

**Comment by happysalada** [comment]
I discovered a small typo in `# Relaod partitions`
in case you are interested in adding it to the PR

**Comment by happysalada** [comment]
Also maybe add a comment about installing `parted` ?

**Comment by happysalada** [comment]
I just thought about this, but how about adding some environment variables at the begining of the script to make clearer what needs to be relplaced.
And then throught the code you only have `$MY_HOSTNAME` or `$MY_SSH_KEY`
I felt the /dev/sda and /dev/sdb need to be replaced also. How about abstracting those to $DISK_1 and $DISK_2?

last solution would be to have arguments for the script.

**Comment by happysalada** [comment]
@asymmetric I now have commit rights, if you are still interested in this, let me know and I can write a detailed set of things to change so we can try to merge this together.
I know this has been a while, so no worries if you have moved on.

**Comment by asymmetric** [comment]
@happysalada congrats on the commit bit!

I think we could merge this one (as it's trivial) and iterate on other (possibly breaking) changes separately.

WDYT?

**Comment by happysalada** [comment]
Agreed, since it's only editing the comments.
I think a better long terme approach would probably be to pass variables to the script.
We can provide this improvement later.

**Comment by happysalada** [comment]
@asymmetric Thank you for your contribution!

**Pull request merged**